packages/sentry/src/sentry.spec.ts

```diff
@@ -161,3 +161,37 @@ describe('reportErrorToSentry', () => {
     });
   });
 });
+
+describe('when using unsafe fields', () => {
+  it('should redact unsafe fields', () => {
+    let safeFields = removeUnsafeFields({
```

Comment:
You can't actually test this through testkit as it doesn't boot sentry it seems. At least `beforeSend` is never invoked. So I renamed things and code to make sense (hopefully).